confident_metrics/metrics.py

```diff
@@ -0,0 +1,161 @@
+import os
+import re
+from threading import Lock
+from typing import Union
+
+from prometheus_client import start_http_server
+from prometheus_client.metrics import MetricWrapperBase
+
+
+_prometheus_server = None
+PROMETHEUS_HOST = os.getenv("PROMETHEUS_HOST", "0.0.0.0")
+PROMETHEUS_PORT = int(os.getenv("PROMETHEUS_PORT", "8000"))
+
+
+class PreciseFloat:
+    """A floating-point value.
```

Comment:
if we have multiline docstring usually we put it as 
```suggestion
    """
    A floating-point value.
```

Let's be consistent and follow it in confident-metrics too. 
(Require to fix all docstrings in PR)

Reply:
Done